src/three/math/OBB.js

```diff
@@ -64,6 +66,13 @@ export class OBB {
 
 		}
 
+		const boxInFrustum = frustum.boxInFrustum( this.orientedBox );
+		if ( ! boxInFrustum ) {
+
+			return false;
+
+		}
+
```

Comment:
Just like we had to make a custom implementation for testing all the OBB points against the frustum planes above -we'll need to do the same thing for the frustum points against the OBB planes here. The "boxInFrustum" function shouldn't be needed in this case.